mj_envs/envs/env_base.py

```diff
@@ -1,16 +1,14 @@
 import os
-
-from gym import error, spaces
-# from gym.utils import seeding
-from gym import utils
-from mj_envs.utils.obj_vec_dict import ObsVecDict
-from mj_envs.robot.robot import Robot
-import numpy as np
-from os import path
+from re import A
```

Comment:
not sure what this is used for?

Reply:
I am not sure either, it was there before, so i just retained it, we can remove it in a future clean up if need be